.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,4 @@
 - [x] I’ve used the pull request template
 - [ ] I’ve written unit tests for these changes
 - [ ] I’ve update the documentation (in `DOCUMENTATION.md`)
-- [ ] I’ve bumped the version number
-    - [ ] in `src/main/resources/application.properties`
-    - [ ] in `src/test/java/uk/gov/service/notify/NotificationClientTest.java::testCreateNotificationClientSetsUserAgent`
-    - [ ] and run the `update_version.sh` script
+- [ ] I’ve bumped the version number (in `src/main/resources/application.properties`)
```

Comment:
Also need to bump the version in the pom.xml file, notifications-java-client/pom.xml:8 will need the new version or use `mvn versions:set -DnewVersion=${version}`

Reply:
Updating the version needs to happen before deploy.sh is run